chapter-09/end/Engine/Particles/EmitterTypes/CircleEmitterType.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+
+namespace chapter_09.Engine.Particles.EmitterTypes
+{
+    public class CircleEmitterType : IEmitterType
+    {
+        public float Radius { get; private set; }
+
+        private RandomNumberGenerator _rnd = new RandomNumberGenerator();
+
+        public CircleEmitterType(float radius)
+        {
+            Radius = radius;
+        }
+
+        public Vector2 GetParticleDirection()
+        {
+            return new Vector2(0f, 0f);
+        }
+
+        public Vector2 GetParticlePosition(Vector2 emitterPosition)
+        {
+            var newAngle = _rnd.NextRandom(0, 2 * 3.14159f);
```

Comment:
I assume you're going for PI here?  System.Math.PI is a handy constant for that

Reply:
> Overall awesome job man!
> 
> Only common thing I noticed was the use of magic numbers instead of using the Texture2D dimensions.

I should probably tighten up the textures so they are the same size as the bounding boxes